test/jdk/java/lang/Class/GetModuleTest.java

```diff
@@ -34,16 +34,13 @@
 
 import jdk.internal.org.objectweb.asm.ClassWriter;
 import static jdk.internal.org.objectweb.asm.Opcodes.*;
-import jdk.internal.misc.Unsafe;
 
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 import static org.testng.Assert.*;
 
 public class GetModuleTest {
```

Comment:
testGetModuleOnVMAnonymousClass is the only test here that uses ASM so you can remove the imports as part of the removal.

Reply:
Hi Alan,
Thanks for find this.  I removed the unneeded imports and \@modules from GetModulesTest.java and updated the comment in ClassAttributesTest.java.
Thanks, Harold

Reply:
Thanks for fixing up these tests, I didn't spot any more in the test/jdk tree.